src/utils/format.ts

```diff
@@ -242,3 +247,36 @@ export function formatSmartMaxPrecision(amount: BigNumber, token?: TokenErc20 |
     smallLimit: getMinimumRepresentableValue(token?.decimals),
   })
 }
+
+/**
+ * Transforms a BigNumber order calculatedPrice (buy/sell) into a string
+ * based on buyToken / sellToken (Erc20 both of them)
+ * e.g:
+ * return: 3,000.2 USDT per WETH
+ *
+ * @param calculatedPrice BigNumber integer amount
+ * @param buyToken Erc20 token
+ * @param sellToken Erc20 token
+ * @param inverted Optional. Whether to invert the price (1/price).
+ */
+
+export function formatCalculatedPriceToDisplay(
+  calculatedPrice: BigNumber,
+  buyToken: TokenErc20,
+  sellToken: TokenErc20,
+  isPriceInverted?: boolean,
+): string {
+  const displayPrice = calculatedPrice.toString(10)
+  const formattedPrice = formatSmart({
+    amount: displayPrice,
+    precision: NO_ADJUSTMENT_NEEDED_PRECISION,
+    smallLimit: HIGH_PRECISION_SMALL_LIMIT,
+    decimals: HIGH_PRECISION_DECIMALS,
+  })
+  const buySymbol = safeTokenName(buyToken)
+  const sellSymbol = safeTokenName(sellToken)
+
+  const [baseSymbol, quoteSymbol] = isPriceInverted ? [sellSymbol, buySymbol] : [buySymbol, sellSymbol]
+
+  return `${formattedPrice} ${quoteSymbol} per ${baseSymbol}`
```

Comment:
Just realized this is inverted.

Not sure though whether it's here or in the input data for the story.

![screenshot_2021-08-20_08-30-47](https://user-images.githubusercontent.com/43217/130257075-ccfcba31-0ceb-4a40-848c-10fc237c20a6.png)

If you are selling `3000 USDT` and buing `1 WETH`, you'll have the price to be `3000 WETH/USDT` (standard notation, can be shortened to just `3000 WETH`) that is the same as `USDT per WETH` (extended notation).

Reply:
If you invert the price, you should have the `0.003... WETH per USDT`